rst/components/holmes-gateway/installation.rst

```diff
@@ -452,53 +271,28 @@ saved to ``config/keys/sources``.
 
 |
 
-----
 
-`Holmes-Frontend <Frontend_>`_
-====================================
+Installation
+#################
 
-TODO
 
+Holmes-Gateway is the endpoint that users interact with when creating tasks for
+Holmes-Totem or Holmes-Totem-Dynamic.
 
+.. code-block:: shell
 
-|
-
-----
-
-`Holmes-Interrogation <Interrogation_>`_
-========================================================================
-
-TODO
-
-
-
-|
-
-----
-
-`Standard Services <Totem_Services_>`_
-========================================================================
-
-TODO
-
-
-
-|
-
-----
-
-`Apache Cassandra <Cassandra_>`_
-========================================================================
-
-TODO
-
+    mkdir -p /data/holmes-gateway
+    cd /data/holmes-gateway
+    git clone https://github.com/HolmesProcessing/Holmes-Gateway.git .
+    go build .
 
+The framework requires one Holmes-Gateway running in Master mode and the Master
```

Comment:
new version of gateway removed the master/slave engines. it is all just one now.